vic/drivers/classic/src/vic_classic.c

```diff
@@ -247,6 +248,13 @@ main(int   argc,
             ******************************************/
 
             for (rec = startrec; rec < global_param.nrecs; rec++) {
+
+                // Set global reference string (for debugging inside vic_run)
+                sprint_dmy(dmy_str, &(dmy[rec]));
```

Comment:
Exactly. The goal here was simply to allow for a more informative error message from inside `vic_run`.  `vic_run` doesn't know anything about the date or the location so this is the drivers opportunity (optional) to provide that.